docs/zh-cn/2.0.2/user_doc/architecture/load-balance.md

```diff
@@ -1,58 +1,52 @@
 ### 负载均衡
-负载均衡即通过路由算法（通常是集群环境），合理的分摊服务器压力，达到服务器性能的最大优化。
 
-### DolphinScheduler-Worker 负载均衡算法
+### Master至Worker 负载均衡算法
```

Comment:
As the demanded by this [page](https://dolphinscheduler.apache.org/en-us/community/development/document.html). It is recommended that spaces be required between Chinese characters and English and numbers, and that no spaces be required between Chinese punctuation marks and English and numbers, to enhance the aesthetics and readability of the mixed Chinese and English layout. The same format applies to the following.

Reply:
I feel your suggestion very much!